pkgs/applications/virtualization/crun/default.nix

```diff
@@ -11,6 +11,7 @@
 , yajl
 , nixosTests
 , criu
+, protobufc
```

Comment:
Please move the crun changes to a separate commit.

Reply:
Also, why is it being added to crun? If it's needed for criu shouldn't it be propagated from there?

Reply:
It is needed by crun with the new criu version but lxd doesn't need it even though it depends on criu? Not sure if this really needs to be propagated from criu.

Reply:
Ah, lxd only uses the binaries and calls them. 

```
In file included from /nix/store/sfg8lcmcgq7fm3p7aafrgdzbl5879rfs-criu-3.17.1-dev/include/criu/criu.h:25,
                 from src/checkpoint.c:30:
/nix/store/sfg8lcmcgq7fm3p7aafrgdzbl5879rfs-criu-3.17.1-dev/include/criu/rpc.pb-c.h:7:10: fatal error: protobuf-c/protobuf-c.h: No such file or directory
    7 | #include <protobuf-c/protobuf-c.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Yeah, I overlooked that the criu headers include protobufc so this needs to be propagated.

Reply:
Unless it is a explicit (and documented) dependency of crun I don't think it should the added there.